README.md

```diff
@@ -19,53 +19,161 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
 
 For the first time, to start all dependent services and Thimble, simply run:
+
 ```
 vagrant up
 ```
-This process can take a while depending on your internet connection speed as it needs to download all dependencies. Once you see a log that says `Express server listening on http://localhost:3500`, you can access Thimble on [http://localhost:3500](http://localhost:3500).
 
-You can now make changes to the Thimble source code on your system and they should be automatically reflected on [http://localhost:3500](http://localhost:3500).
+Depending on your internet connection speed, this process can take a while (Since it needs to download all dependencies not listed above).
+Once you see `Express server listening on http://localhost:3500`, you are ready to start using Thimble on [http://localhost:3500](http://localhost:3500)!
+Any changes made to the Thimble source code on your system will automatically be reflected on [http://localhost:3500](http://localhost:3500).
 
-To stop running Thimble, simply press `Ctrl+C` twice.
+To stop Thimble, simply press `Ctrl+C` twice.
+
+To restart Thimble, run:
 
-To restart Thimble again, run:
 ```
 vagrant reload --provision
 ```
-This will take a much shorter time to setup compared to the `vagrant up` command.
+
+This will take less time to setup compared to the `vagrant up` command.
+
+##1b) Manual Installation
+You can also setup Thimble and it's needed components outside Vagrant and Virtualbox. This might be needed if you want to:
+- Host your own instance of Thimble
+- Cannot run virtualization software on your computer
+
+### Pre-Requisites for Manual Installation
+In order for Thimble to be installed correctly, the following dependencies needs to be installed:
+
+- Node.js 4.x or above (see note below)
+  - * **Note:** The login.webmaker.org dependency needs a node version of 4.x only while all the other dependencies work with a node version of 4.x and above. We suggest installing [NVM](https://github.com/creationix/nvm) to allow the use of multiple versions of node.
+- [Brackets (Bramble)](#installing-brackets-bramble)
+- [Webmaker ID server](#idwebmakerorg)
+- [Webmaker Publishing Server](#publishwebmakerorg)
+- [Postgresql 9.4 or above (for the publish.webmaker.org dependency)](#postgresql)
+- g++ 4.8 or above (for the login.webmaker.org dependency)
+- [Webmaker Login Server](#loginwebmakerorg)
+
+The following is an abbreviated guide to getting it all set up. Please see each server's README for more details.
+
+### Manually Installing the Parts
+Please note: On Windows, use ``copy`` instead of ``cp``
+
+####Thimble
+* Fork and clone https://github.com/mozilla/thimble.mozilla.org
+* Run ``cp env.dist .env`` to create an environment file
+* Run ``npm install`` to install dependencies
+* Run ``npm start`` to start the server
+* Once everything is ready and running, Thimble will be available at [http://localhost:3500/](http://localhost:3500/)
```

Comment:
let's move this line after the `publish.webmaker.org` section as a general statement.